evm.md

```diff
@@ -166,8 +166,9 @@ Our semantics is modal, with the initial mode being set on the command line via
 -   `VMTESTS` skips `CALL*` and `CREATE` operations.
 
 ```k
-    syntax Mode ::= "NORMAL"  [klabel(NORMAL), symbol]
-                  | "VMTESTS" [klabel(VMTESTS), symbol]
+    syntax Mode ::= "NORMAL"   [klabel(NORMAL), symbol]
+                  | "VMTESTS"  [klabel(VMTESTS), symbol]
+                  | "COVERAGE" [klabel(COVERAGE), symbol]
```

Comment:
Can we declare this in `analysis.md` instead of `evm.md`? `evm.md` is meant to be the semantics of EVM only (as much as possible).

Reply:
@ana-pantilie Just to make sure, you can append the new syntax declaration, by having the following:
```
syntax Mode ::= "COVERAGE" ...
```